src/modules/core/producer_timewarp.yml

```diff
@@ -52,6 +52,14 @@ parameters:
     required: yes
     mutable: no
 
+  - identifier: warp_pitch_compensate
```

Comment:
I would be open to comment on this name. Feels verbose.
It has the "warp" prefix because the producer adopts all the properties of the encapsulated producer, and I do not want there to be a conflict.

Reply:
I would be ok with simply `warp_pitch`

Reply:
agree. Done.